KsApi/GraphSchema.swift

```diff
@@ -192,17 +193,18 @@ public enum Query {
     }
   }
 
-  public enum User {
+  public indirect enum User {
```

Comment:
Is there a reason why we are enabling indirection for all cases?

Reply:
I think this is because of the recursive relationship between `Project` <-> `User`. Without this it gives this error:
<img width="349" alt="image" src="https://user-images.githubusercontent.com/3735375/75481116-659c7280-5957-11ea-8f30-a5c06fc57f4f.png">